host/libhackrf/src/hackrf.c

```diff
@@ -2202,6 +2290,42 @@ int ADDCALL hackrf_set_operacake_ranges(hackrf_device* device, uint8_t* ranges,
 	}
 }
 
+static uint8_t dwell_data[5*16];
```

Comment:
maybe use HACKRF_OPERACAKE_MAX_DWELL_TIMES here for clarity

Reply:
also maybe sizeof(hackrf_operacake_dwell_time)?

Reply:
I was reluctant to use sizeof(hackrf_operacake_dwell_time) there because I think it may differ from the size of the control transfer, depending on the packing choices of different compilers/hosts.
I could add a local define there just to make it clearer?